<!DOCTYPE html>
<html lang="en">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
    integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7"
    crossorigin="anonymous">

<head>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>Hiram Genomics Store</title>
</head>

<body>
    <div class="container">
        <div class="row hidden-md hidden-lg">
            <div class="col-md-12">
                <h3><a href="/">Hiram Genomics Store</a></h3>
            </div>
        </div>
        <div class="row">
            <div class="hidden-xs hidden-sm">
                <div class="col-md-2">
                    <div class="row">
                        <div class="col-md-12">
                            <img src="../../img/logo.jpg" class="img-responsive" alt="Store Logo">
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-med-12">
                            <p class="lead">Research creativity in your classroom.</p>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-md-6"></div>
                        <div class="col-md-6">
                            <h4><a href="/hiramgenomicsstore/">Back</a></h4>
                        </div>
                    </div>
                </div>
            </div>
            <div class="col-md-10 col-sm-12 col-xs-12">
                <h1>Identify a Microbe:</h1>
                <h2>Using DNA Sequence (16S rRNA gene) for Strain Identification</h2>
                <p class="lead">This primer set amplifies a roughly 1450 bp fragment of a gene encoding
                    the 16S rRNA gene from all members of the domain Bacteria (Lane,
                    1991; Stackebrandt &amp; Liesack, 1993). All organisms on
                    Earth have one or more 16S/18S rRNA genes, but this primer set
                    was originally shown to be specific for the domain Bacteria.
                    However, recent work has shown that these primers can also amplify
                    the 18S rRNA gene from certain corals (Galkiewicz &amp; Kellogg,
                    2008). More importantly, the original 27F primer misses some
                    key subgroups in the domain Bacteria, so we use the 27F-YM+3/1492R
                    primer mix to optimize strain coverage (Frank et al., 2008).
                    The annealing temperature for this PCR reaction is 48C. This
                    primer set can be purchased separately or as part of a kit that
                    includes colony DNA isolation and sequencing of PCR amplicons.</p>
                <h2>Products:</h2>
                <table class="table table-hover">
                    <tr>
                        <td>16S rRNA Gene Primer Set.</td>
                        <td>Enough for 10 PCR reactions</td>
                        <td>$15 + S/H</td>
                    </tr>
                    <tr>
                        <td> Identify-a-Microbe Kit reagents</td>
                        <td>Enough for 10 DNA isolations, PCR reactions, &amp; DNA
                            sequencing (sent out)</td>
                        <td>$140 + S/H</td>
                    </tr>
                </table>
                <h3>Bacteria are Everywhere</h3>
                <p>The famous paleontologist and science writer Stephen J. Gould once
                    wrote that there has never been an "age of insects", "age of
                    dinosaurs", or "age of mammals". There has really only been an
                    "age of bacteria" because these relatively tiny organisms have
                    dominated the earth since its earliest biotic beginnings in so
                    many ways (Gould, 1996). By "little b" bacteria, Gould meant
                    prokaryotes of which there are two kinds. The domain Bacteria
                    includes the bacteria we all know - the gut inhabitant and sometimes
                    pathogen E. coli, Lactobacillus species involved in yogurt and
                    cheese production, and Streptococcus pyogenes that can cause
                    strep throat, just to name a few. The domain Archaea is a very
                    different group of prokaryotes and we didn't even know they existed
                    as a distinct group until the late 1970's, but it turns out they
                    are also ubiquitous (Woese &amp; Fox, 1977). We find bacteria
                    in all kinds of recognizable habitats, oceans, lakes, and soil,
                    but also in strange places like glaciers, hot springs, deep sea
                    thermal vents, and on or inside other organisms. They be found
                    at temperatures from 0oC to over 100oC, at pH's from &amp;lt;
                    1 to&amp;gt;10, at great pressures, at salt concentrations so
                    high the salt starts to fall out of solution, and across a broad
                    range of nutrient concentrations. In many of these places, we
                    can also find some eukaryotic microbes - algae, fungi, and others.
                    Yet, we are still left with our question of "how do we know who
                    they are?"</p>
                <p>For bacteria, the choice for decades has been plating samples out
                    on various culture media and then taking isolated colonies and
                    restreaking them on fresh plates to ensure that each colony is
                    a pure culture of one particular bacterial strain. If it is a
                    pure culture, then every colony on the restreaked plate will
                    look the same as each other. Once we are sure that a particular
                    strain is pure, colonies of that strain can then be used to identify
                    it using a variety of biochemical assays (e.g., can it break
                    down urea), stains (e.g., the very popular Gram stain), and growth-based
                    tests (e.g., can the strain grow in very salty conditions such
                    as the presence of 10% NaCl).</p>
                <p>Sometimes, we may be interested in a particular bacterial strain,
                    but we cannot use the standard culture-based lab methods - we
                    cannot figure out how to get that strain to grow up on a plate
                    or in a broth culture. Evidence has accumulated during the last
                    two decades that cultured-based identification methods are not
                    good estimators of microbial diversity because they catch only
                    a small percentage (0.5-5%) of the microbial diversity present
                    in any habitat at any time (Staley &amp; Konopka, 1985).
                    Even after more than a century of efforts, we only can get a
                    minority of bacteria to grow in culture (Kaeberlein, Lewis, &amp;
                    Epstein, 2002). Does that mean we cannot identify unculturable
                    bacteria? No, we just have to either deal with them as part of
                    an overall community (see our Metagenomic DNA Analysis kits)
                    or we have to find a way to obtain a "mostly pure" sample of
                    the bacterial strain of interest where the vast majority of the
                    organisms are the same kind. This sometimes happens in nature
                    or we can enrich for it (e.g., Winogradsky column).</p>
                <p>Regardless of how we obtain a particular bacterial strain, by culture,
                    straight from nature, or from enrichment, how can we identify
                    that strain without having to assume that it is culturable? A
                    genetics tool comes to the rescue! We typically think of genetics
                    as just a set of concepts (e.g., gene, chromosome) and processes
                    (e.g., meiosis, recombination), but genetics is also an experimental
                    toolbox that can be used to address questions throughout the
                    life sciences. One of the tools in our genetics toolbox is a
                    technique called Polymerase Chain Reaction or PCR (Sakai et al.,
                    1985). In PCR, a stretch of DNA is replicated over and over and
                    over again until you have a workable quantity of it. The specificity
                    in PCR comes from the choice of short stretches of single-stranded
                    DNA called primers onto which DNA Polymerase adds nucleotides
                    during DNA replication. Those primers might be specific to a
                    DNA sequences found only in a species, a genus, a family, or
                    some higher level taxon (e.g., Blackwood, Oaks, &amp; Buyer,
                    2005). Any gene is a potential PCR target; it just depends on
                    the question you are addressing.</p>
                <p>Since we don't know the identity of our unknown bacterial strain,
                    we will use PCR primers targeted to a gene found in all organisms
                    - the 16S/18S rRNA gene. This gene encodes one of the rRNA molecules
                    that are used to construct ribosomes, the essential cell machinery
                    that catalyzes the formation of new proteins. The 16S/18S rRNA
                    gene is highly conserved across all life on Earth, meaning that
                    if you compare the gene sequence from two different organisms
                    you will be easily able to tell that it is pretty much the same
                    along its length. However, it is not 100% identical. Rather,
                    the gene sequence has slowly changed over evolutionary time -
                    in other words, from the moment one group of organisms is split
                    into two and they don't reproduce or exchange DNA with each other
                    anymore, then mutations will slowly accumulate in the 16S/18S
                    rRNA gene such that the two groups are now slightly different
                    in their DNA sequences. The longer the time since the divergence
                    of the group, the more differences between their DNA sequences.
                    This is the basic idea behind using DNA and protein sequences
                    to understand evolutionary relationships and is the basis for
                    the currently accepted tree of life (Woese &amp; Fox, 1977).
                    These differences in gene sequence can also used to identify
                    unknown organisms by comparing their 16S/18S rRNA gene sequence
                    to a database of sequences from known organisms. Certain parts
                    of the 16S/18S rRNA gene mutate faster than others and that dictates
                    what regions are used for PCR amplification. With PCR, you don't
                    have to grow the organisms and you don't have to have the entire
                    genome intact. As long as you can amplify a portion of the 16S/18S
                    rRNA gene and determine its sequence, you have a chance to identify
                    the organism.</p>
                <h3>Starting Ideas for Projects:</h3>
                <ul>
                    <li>
                        <p>Your students can identify some habitat of interest, be it
                            commonplace, rare, or weird, and sample the habitat and
                            try to grow up some of its microbial community members
                            on culture plates. Each isolated colony on a plate is
                            a potential study subject. We would recommend first streaking
                            a colony for pure culture on a new plate, but then your
                            students can isolate genomic DNA from a pure colony and
                            use PCR to amplify a portion of the 16S/18S rRNA gene.
                            Then we will help you get the sequence of that amplified
                            DNA fragment and you can finally compare that sequence
                            to database of known sequences to potentially identify
                            your unknown organism.</p>
                    </li>
                </ul>
                <h3>References</h3>
                <p>Blackwood, C.B., A. Oaks, &amp; J.S. Buyer, 2005. Applied Environ.
                    Micro. 71:6193-8. Phylum- and class-specific PCR primers for
                    general microbial community analysis.</p>
                <p>Frank, J.A., C.I. Reich, S. Sharma, J.S. Weisbaum, B.A. Wilson, &amp;
                    G.J. Olsen, 2008. Applied Environ. Micro. 74:2461-70. Critical
                    evaluation of two primers commonly used for amplification of
                    bacterial 16S rRNA genes.</p>
                <p>Galkiewicz, J.P., &amp; C.A. Kellogg, 2008. Applied Environ. Micro.
                    74:7828-31. Cross-kingdom amplification using Bacteria-specific
                    primers: complications for studies of coral microbial ecology.</p>
                <p>Gould, 1996. Full House, pp. 175-192, New York: Harmony Books.</p>
                <p>Kaeberlein, Lewis, &amp; Epstein, 2002. Science 296:1127-9. Isolating
                    "uncultivable" microorganisms in pure culture in a simulated
                    natural environment.</p>
                <p>Lane, D. J. 1991. p. 115-175. In Nucleic Acid Techniques in Bacterial
                    Systematics.
                </p>
                <p>Lane, D. J. 1991. 16S/23S rRNA sequencing, p. 115-147. In E. Stackebrandt
                    &amp; M. Goodfellow (ed.), Nucleic acid techniques in bacterial
                    systematics. John Wiley &amp; Sons, New York, NY.</p>
                <p>Saiki et al., 1985. Science 230:1350-4. Enzymatic amplification of
                    beta-globin genomic sequences and restriction site analysis for
                    diagnosis of sickle cell anemia.</p>
                <p>Stackebrandt, E., and W. Liesack. 1993. Nucleic acids and classification,
                    p. 152-189. In M. Goodfellow &amp; A. G. O’Donnell (ed.), Handbook
                    of new bacterial systematics. Academic Press, London, England.</p>
                <p>Staley &amp; Konopka, 1985. Annual Rev. Micro. 39: 321-46. Measurement
                    of in situ activities of nonphotosynthetic microorganisms in
                    aquatic and terrestrial habitats.</p>
                <p>Woese &amp; Fox, 1977. Proc. Natl. Acad. Sci. USA 74:5088-90. Phylogenetic
                    structure of the prokaryotic domain: the primary kingdoms.</p>
            </div>
</body>

</html>
