<!DOCTYPE html>
<html lang="en">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
    integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7"
    crossorigin="anonymous">

<head>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>Hiram Genomics Store</title>
</head>

<body>
    <div class="container">
        <div class="row hidden-md hidden-lg">
            <div class="col-md-12">
                <h3><a href="/hiramgenomicsstore/">Hiram Genomics Store</a></h3>
            </div>
        </div>
        <div class="row">
            <div class="hidden-xs hidden-sm">
                <div class="col-md-2">
                    <div class="row">
                        <div class="col-md-12">
                            <img src="../../img/logo.jpg" class="img-responsive" alt="Store Logo">
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-med-12">
                            <p class="lead">Research creativity in your classroom.</p>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-md-6"></div>
                        <div class="col-md-6">
                            <h4><a href="/">Back</a></h4>
                        </div>
                    </div>
                </div>
            </div>
            <div class="col-md-10 col-sm-12 col-xs-12">
                <h1>Metagenomics DNA Analysis:<br />
                Using DNA to Characterize a Microbial Community</h1>
                <p class="lead">This kit enables a class of up to 28 students to isolate metagenomic
                    DNA from environmental samples through PCR using primers of your
                    choice, and analyze your results using agarose gel electrophoresis.
                    Your class designs the experiment, desides on the focal group
                    of the organisms and carries out the experiment.</p>
                <p><b>Products</b></p>
                <table class="table">
                    <tr>
                        <td>Metagenomics DNA Analysis Kit</td>
                        <td>reagents for metagenomic DNA isolation from 7 samples, PCR
                            reactions, &amp; gel electrophoresis $140 + S/H</td>
                    </tr>
                </table>
                <p><b>Who Lives There?</b></p>
                <p>As you walk along a stream, you notice an orange plume coming out
                    of the rocky bank into the stream. "Is that chemical natural
                    or man-made" is probably the first thing you ask yourself. However,
                    you may well want to ask "who lives there" because chances are
                    there are lots of microbes present and their activities may be
                    involved in the formation of the orange plume. Microbes are everywhere,
                    both prokaryotic (DNA not organized in a nucleus) and eukaryotic
                    (DNA organized in a nucleus), but how do we figure out who they
                    are? That is the question this research kit helps you and your
                    students to answer.</p>
                <p>Microbial members of all 3 domains of life, Archaea, Bacteria, and
                    Eucarya, live in all kinds of recognizable habitats such as oceans,
                    lakes, and soil, but also in strange places like glaciers, hot
                    springs, deep sea hydrothermal vents, and on and inside other
                    organisms (DeLong, 1998; Pikuta et al., 2007; Rappe &amp; Giovannoni,
                    2003). They can be found at temperatures from 0oC to over 100oC,
                    at pH’s from &lt; 1 to&gt;10, at great pressures, at salt concentrations
                    so high the salt starts to fall out of solution, and across a
                    broad range of nutrient concentrations. Yet, we are still left
                    with our question of how do we know who they are and this seemingly
                    simple question is crucial to ecology.</p>
                <p>Ecologists ask questions about individual organisms as well as populations,
                    how they interact with their environment and with other organisms.
                    Central to answering many ecological questions is the ability
                    to count or estimate the number of individuals of a given species.
                    If we are talking elephants or mature white oak trees, then it
                    may not be that hard a task. If we have to determine the number
                    of blue jays or spring peeper frogs, it won’t be as easy and
                    our best bet is an estimate based on sampling. Yet, what if we
                    are interested in microbes? For some, such as diatoms or ciliate
                    protozoans, we could look at samples under the microscope although
                    this will take some background knowledge of how to distinguish
                    various taxa based on morphology. For bacteria, the choice for
                    decades has been plating samples out on various culture media
                    and counting colonies based on the assumption that each colony
                    came from a single cell that was laid down at that spot. Colonies
                    can then be identified using a variety of biochemical assays,
                    stains, and growth-based tests. However, evidence has accumulated
                    during the last two decades that cultured-based identification
                    methods are not good estimators of microbial diversity because
                    they catch only a small percentage (0.5-5%) of the microbial
                    diversity present in any habitat at any time (Staley &amp; Konopka,
                    1985). Even after a century plus of efforts, we only can get
                    a minority of bacteria to grow in culture (Kaeberlein, Lewis,
                    &amp; Epstein, 2002). Given this problem, how can one detect
                    the presence of a microorganism without growing it? Microscopy
                    is possible, but it cannot separate live from dead cells nor
                    can it identify strains to any fine degree by itself. However,
                    microscopy using fluorescently-labeled DNA probes can identify
                    strains to varying degrees based on sequence homology (DeLong,
                    Wickham, &amp; Pace, 1989). A genetics tool comes to the rescue!
                    We typically think of genetics as just a set of concepts (e.,g.,
                    gene, chromosome) and processes (e.,g., meiosis, recombination),
                    but genetics is also an experimental toolbox that can be used
                    to address questions throughout the life sciences. Those questions
                    may deal with medicine, agriculture, development, behavior, and
                    ecology, yes ecology.</p>
                <p>Another tool in our genetics toolbox is the technique Polymerase
                    Chain Reaction or PCR (Sakai et al., 1985). PCR allows us to
                    amplify specific sequences. The specificity of recognition in
                    PCR comes from the choice of PCR primers that are used to start
                    new DNA synthesis in each PCR cycle. The primers might be specific
                    to a particular species, a genus, a family, or some higher level
                    taxon (e.g., Blackwood, Oaks, &amp; Buyer, 2005). Any gene is
                    a potential PCR target, it just depends on the question you are
                    addressing.
                </p>
                <p><b>Research in Action in Your Classroom</b></p>
                <p>Let us imagine for the moment that you are interested in the bacterial
                    genus Salmonella which is in the news more often these days due
                    to illness outbreaks usually linked to contaminated food supplies.
                    You want to know if store bought chicken products carry Salmonella,
                    but how can you answer that question? PCR could answer the question
                    for if you had a DNA sample to test. Since you may not be able
                    to grow Salmonella easily in your lab (or may not want to for
                    safety reasons), it would be much easier to look for Salmonella-specific
                    DNA sequences directly. As an alternative to culturing, you can
                    swab the chicken products to collect any microbes present and
                    isolate total DNA immediately. When one isolates DNA from all
                    the microbes present in an environmental sample, this is called
                    a metagenomic DNA sample because it represents all the microbes
                    and their genomes (all their DNA). We can then carry out PCR
                    on the metagenomic DNA sample using the Salmonella-specific PCR
                    primers. Finally, we can detect the PCR products using agarose
                    gel electrophoresis, a standard DNA lab technique.</p>
                <p><b>Starting Ideas for Projects</b></p>
                <p>Our initial line of PCR primers include those for E. coli, Salmonella,
                    the domain Archaea, methanogenic members of the Archaea, and
                    halophilic members of the Archaea. Using one of these primer
                    sets, you could ask one of following questions but really you
                    are only limited by your own curiosity.</p>
                <ol>
                    <li> Look for potential E. coli contamination (as a indicator of fecal
                        contamination) of local waterways.</li>
                    <li>Look for Salmonella on kitchen countertops.</li>
                    <li>Look for Archaea in everyday habitats in your house, your yard,
                        soil, streams, ponds, etc.</li>
                </ol>
                <p>We do have many other PCR primers available through our customization
                    process, so let your curiosity soar.</p>
                <p><b>References for More Information</b></p>
                <p>Blackwood, Oaks, &amp; Buyer, 2005. Applied Environ. Micro. 71:6193–8.
                    Phylum- and class-specific PCR primers for general microbial
                    community analysis.
                    <p>DeLong, Wickham, &amp; Pace, 1989. Science 243:1360-3. Phylogenetic
                        stains: ribosomal RNA-based probes for the identification
                        of single cells.</p>
                    <p>DeLong, 1998. Everything in moderation: Archaea as 'non-extremophiles'.
                        Current Opinion in Genetics &amp; Development 8:649-54.</p>
                    <p>Pikuta, Hoover, &amp; Tang, 2007. Microbial extremophiles at
                        the limits of life. Critical Reviews in Microbiology 33:183-209.</p>
                    <p>Kaeberlein, Lewis, &amp; Epstein, 2002. Science 296:1127–9. Isolating
                        "uncultivable" microorganisms in pure culture in a simulated
                        natural environment.</p>
                    <p>Rappe &amp; Giovannoni, 2003. Annual Rev. Micro. 57: 369-94.
                        The uncultured microbial majority.</p>
                    <p>Saiki et al., 1985. Science 230:1350-4. Enzymatic amplification
                        of beta-globin genomic sequences and restriction site analysis
                        for diagnosis of sickle cell anemia.</p>
                    <p>Staley &amp; Konopka, 1985. Annual Rev. Micro. 39: 321-46. Measurement
                        of in situ activities of nonphotosynthetic microorganisms
                        in aquatic and terrestrial habitats.
                    </p>
            </div>
</body>

</html>
