<!DOCTYPE html>
<html lang="en">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
    integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7"
    crossorigin="anonymous">

<head>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>Hiram Genomics Store</title>
</head>

<body>
    <div class="container">
        <div class="row hidden-md hidden-lg">
            <div class="col-md-12">
                <h3><a href="/">Hiram Genomics Store</a></h3>
            </div>
        </div>
        <div class="row">
            <div class="hidden-xs hidden-sm">
                <div class="col-md-2">
                    <div class="row">
                        <div class="col-md-12">
                            <img src="../../img/logo.jpg" class="img-responsive" alt="Store Logo">
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-med-12">
                            <p class="lead">Research creativity in your classroom.</p>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-md-6"></div>
                        <div class="col-md-6">
                            <h4><a href="/">Back</a></h4>
                        </div>
                    </div>
                </div>
            </div>
            <div class="col-md-10 col-sm-12 col-xs-12">
                <img src="../img/metagenomic-dna-datasets.jpg" class="img-responsive" alt="Metagenomic DNA Datasets">
                <p><b>Who Lives in That Place?</b></p>
                <p>Ecologists ask questions about individual organisms as well as populations,
                    how they interact with their environment and with other organisms
                    of their same and/or other species. Central to answering many
                    ecological questions is the ability to count or estimate the
                    number of individuals of a given species. If we are talking elephants
                    or mature white oak trees, then it may not be that hard a task.
                    If we have to determine the number of blue jays or spring peeper
                    frogs, it won’t be as easy and our best bet is an estimate based
                    on sampling. Yet, what if we are interested in microbes, both
                    prokaryotic and eukaryotic? For some, such as diatoms or ciliate
                    protozoans, we could look at samples under the microscope although
                    this will take some background knowledge of how to distinguish
                    various taxa based on morphology. For bacteria, the choice for
                    decades has been plating samples out on various culture media
                    and counting colonies based on the assumption that each colony
                    came from a single cell that was laid down at that spot. Colonies
                    can then be identified using a variety of biochemical assays,
                    stains, and growth-based tests.</p>
                <p>Evidence has accumulated during the last two decades that cultured-based
                    identification methods are not good estimators of microbial diversity
                    because they catch only a small percentage (0.5-5%) of the microbial
                    diversity present in any habitat at any time. Even after a century
                    plus of efforts, we only can get a minority of bacteria to grow
                    in culture. Given this problem, how can one detect the presence
                    of a microorganism without growing it? Modern genetics gives
                    us a new option – look at the DNA of all the organisms present
                    in a microbial community.</p>
                <p>To sample microbes in an unbiased manner, we typically use PCR (Polymerase
                    Chain Reaction) and primers targeted to a gene found in all organisms
                    – the 16S/18S rRNA gene. This gene is highly conserved, but it
                    has slowly changed over evolutionary time and those changes are
                    the basis of the currently accepted tree of life. They are also
                    used to identify organisms. Certain parts of the 16S rRNA gene
                    mutate faster than others and that dictates what regions are
                    used for PCR amplification. With PCR, you don’t have to grow
                    the organisms and you don’t have to have the entire genome intact.
                    As long as you can amplify a small region of the 16S rRNA gene
                    and determine its sequence, you have a chance to identify the
                    organism. Easy enough if you start with a pure culture, but we
                    are talking about sampling the unknown. If you apply the 16S
                    rRNA gene PCR approach to an unknown assemblage of uncultured
                    microbes, this is called metagenomics. Because you are isolating
                    DNA from a mixed population, your PCR amplification product will
                    be a mixture. The only way to get knowledge of specific sequences
                    is to separate them from each other, usually by recombinant DNA
                    cloning. More recently, scientists have begun to gather additional
                    metagenomics data sets either by using PCR primers for other
                    well-conserved genes or by sequencing random fragments in a large-scale
                    fashion.
                </p>
                <p>While you and your students can isolate metagenomic DNA from your
                    own environmental samples and look for specific groups of organisms
                    by PCR using our Metagenomics PCR Kits, Hiram Genomics Store
                    also offers several large datasets of previously sequenced 16S
                    rRNA gene sequences from different experiments. What follows
                    is a brief description of two such experiments.</p>
                <p><b>Example 1: The Microbial Community of the Human Armpit &amp; the Impact of Deodorant</b></p>
                <p>Our skin is a complex and variable habitat for microbes. While skin
                    is composed of many layers, microbes only inhabit the outer surface
                    on healthy individuals. The skin has factors such as variable
                    temperature, moisture, and lipid content, high salt concentration,
                    keratin (waterproofs the skin), low pH, and antimicrobial peptides
                    that help protect against invasion. Some bacteria, such as Staphylococcus
                    and Streptococcus, have evolved to survive in such an environment
                    and may cause disease if a portal of entry presents itself. The
                    subject of our investigation, the human armpit or axilla, is
                    on the higher end of the warmth and moisture spectrum compared
                    to the rest of the skin, analogous to tropical rainforests as
                    compared to other biomes on Earth.</p>
                <p>Our study volunteer was a 50-year-old male in good health who had
                    not taken antibiotics for several months. The axilla were washed
                    with water only for 10 days (no soap) prior to sampling. No deodorant
                    was applied to the left axilla during that time, while deodorant
                    was applied to the right axilla. The deodorant used contained
                    propylene glycol as the potential active ingredient. At the end
                    of 10 days, each axilla was rubbed with sterile swabs wetted
                    with sterile 0.9% NaCl. The swabs were vortexed in 1 ml 0.9%
                    NaCl to knock off bacteria, the cells pelleted, and total metagenomic
                    DNA isolated. 16S rRNA gene sequences were amplified from each
                    metagenomic DNA sample and subjected to next generation DNA sequencing.
                    Your students can now analyze the sequence data to determine
                    who lives on the surface of the human armpit and the impact of
                    one type of deodorant on that microbial community.</p>
                <p><b>Example 2: How Nutrients &amp; Salinity Impact Marine Microbial Communities</b></p>
                <p>
                    The experiment started in the summer of 2009 with beach sand and seawater from the
                    central Gulf coast of Florida (Anna Maria Island). A mixture
                    of 100 ml of beach sand and 100 ml of sterile sand were added
                    to each of 3 reused plastic bottles (washed and sterilized with
                    ethanol) along with 4 g Na2CO3 (source of CO2 for autotrophs),
                    4 g MgSO4 (source of sulfate for sulfur redox cycle), and 2 g
                    cellulose (complex carbohydrate for heterotrophs). One bottle
                    was left as is (Bottle E). The other two bottles (A and B) also
                    received some mineral salts solution (source of N, P, K) to mimic
                    nutrient runoff from agriculture or human waste. To see the potential
                    impact of evaporation in an isolated tidal pool on the seashore,
                    29.22 g NaCl was added to Bottle B only to mimic the increase
                    in salinity upon evaporation. Finally, a mixture of 2/3 seawater
                    and 1/3 sterile water was used to fill up each of the bottles
                    to about 1 inch from the top. Light was provided by continual
                    exposure to cool white fluorescent lights (8000 lux). The pictures
                    below were taken at 4 time points after set up. At the last time
                    point, each bottle was briefly mixed to dislodge bacteria from
                    sediment particles and from the sides of the bottle, the sediment
                    was allowed to settle briefly, and the liquid transferred into
                    centrifuge tubes. The bacteria were then pelleted and total metagenomic
                    DNA was isolated. 16S rRNA gene sequences were amplified from
                    each metagenomic DNA sample and subjected to next generation
                    DNA sequencing. Now your students can analyze the sequence data
                    and determine which groups of bacteria are thriving under each
                    condition and then use the scientific literature to develop hypotheses
                    that might explain the data.</p>
            </div>
</body>

</html>
